### PR TITLE
Handle Compat ShareElement

### PR DESCRIPTION
**Pull request by Judrummer**
- handle sharedElementcallback
- handle make sharedElement

### CODE REVIEW

File: kxandroid-support-v4/src/main/java/com/taskworld/kxandroid/support/v4/FragmentActivities.kt [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+import android.os.Parcelable
+import android.support.v4.app.FragmentActivity
+import android.support.v4.app.SharedElementCallback
+import android.view.View
+
+/**
+ * Created by tipatai on 4/26/16 AD.
+ */
+
+class KxSharedElementCallback {
```
Comment (kittinunf) [inline kxandroid-support-v4/src/main/java/com/taskworld/kxandroid/support/v4/FragmentActivities.kt]: I think you'd better going something along this line. So, it is consistent.

https://github.com/taskworld/KxAndroid/blob/master/kxandroid/src/main/java/com/taskworld/kxandroid/TextViews.kt#L37

File: kxandroid-support-v4/src/main/java/com/taskworld/kxandroid/support/v4/FragmentActivities.kt [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    var onSharedElementStart: ((MutableList<String>?, MutableList<View>?, MutableList<View>?) -> Unit)? = null
+    var onSharedElementEnd: ((MutableList<String>?, MutableList<View>?, MutableList<View>?) -> Unit)? = null
+    var onMapSharedElements: ((MutableList<String>?, MutableMap<String, View>?) -> Unit)? = null
+    var onRejectSharedElements: ((MutableList<View>?) -> Unit)? = null
+    var onCreateSnapshotView: ((Context?, Parcelable?) -> Unit)? = null
+    var onCaptureSharedElementSnapshot: ((View?, Matrix?, RectF?) -> Unit)? = null
+
+}
+
+fun FragmentActivity.kx_setExitSharedElementCallback(callbackInit: KxSharedElementCallback.() -> Unit) {
```
Comment (kittinunf) [inline kxandroid-support-v4/src/main/java/com/taskworld/kxandroid/support/v4/FragmentActivities.kt]: As mention above.

File: kxandroid-support-v4/src/main/java/com/taskworld/kxandroid/support/v4/Views.kt [hunk 3]
```diff
@@ -1,5 +1,6 @@
 package com.taskworld.kxandroid.support.v4
 
+import android.support.v4.util.Pair as PairV4
```
Comment (kittinunf) [inline kxandroid-support-v4/src/main/java/com/taskworld/kxandroid/support/v4/Views.kt]: Why do we use Pair from Android?

Comment (Judrummer) [inline kxandroid-support-v4/src/main/java/com/taskworld/kxandroid/support/v4/Views.kt]: ActivityOptionsCompat.makeSceneTransitionAnimation need android.support.v4.util.Pair as parameters

**Comment by kittinunf** [comment]
I think it is good to go. @onlydevtwice can you confirm this?

**Pull request merged**